### ISSUE TITLE
npx task remains stuck

### ISSUE THREAD

**Issue by Shampra** [SamBouwer/any-docker#20]
`npx electron-builder@24.6.3 install-app-deps --platform=win32 --arch=x64` never finishes

```
Building Windows client...
  • electron-builder  version=24.6.3
  • loaded configuration  file=package.json ("build" field)
  • rebuilding native dependencies  dependencies=keytar@7.9.0 platform=win32 arch=x64
```
npx  seems to have this problem sometimes. Using npm exec, it works

**Comment by SamBouwer** [comment]
Merged

**Comment by rotu** [comment]
Root cause https://github.com/npm/cli/issues/6662 is now fixed in [npm@10.1.0](https://github.com/npm/cli/releases/tag/v10.1.0)!

**Comment by SamBouwer** [comment]
> Root cause https://github.com/npm/cli/issues/6662 is now fixed in [npm@10.1.0](https://github.com/npm/cli/releases/tag/v10.1.0)!
> 

Thanks! I'll update the scripts and test. I'll need to add some info about npm/node version depencies anyway :)

**Comment by SamBouwer** [comment]
> Root cause [npm/cli#6662](https://github.com/npm/cli/issues/6662) is now fixed in [npm@10.1.0](https://github.com/npm/cli/releases/tag/v10.1.0)!

This indeed seems to allow the use of npx instead of npm exec. Thanks again for mentioning :)
